### DISCUSSION TITLE
Reduce the number of shrads for so-zeek index?

### DISCUSSION THREAD

**Discussion by ben-sec** [Security-Onion-Solutions/securityonion#6735] [Unsupported Versions] [answered]
Hello!
After the [outage of my search node](https://github.com/Security-Onion-Solutions/securityonion/discussions/6684) I'm wondering, if you can recommend to reduce the number of shrads for so-zeek indices (default of 5) in order to lower the overall number of shrads. Currently I have 93 so-zeek indices (ranging from 5 to 25 GB in size) totaling 465 shrads (of 938 overall shrads count . Or what is the reason why the so-zeek index gets 5 instead of only 1 shrad like all other indices?

My goal is to extend the log retention to at least 180 days and my search node crashed at about 150 days because I reached the shard limit of 1000. So far I read that it's not recommended to go beyond 1000 shards per node. Or do I have to install an additional ES instance? Any advise is highly appreciated. Thanks in advance!

Cheers, Ben.

**Comment by dougburks** [comment, marked as answer]
Yes, you can reduce your shard count for so-zeek as long as your shard size is under 50GB:
https://docs.securityonion.net/en/2.3/elasticsearch.html#shards
